test/helpers/utils.js

```diff
@@ -26,3 +27,10 @@ exports.checkExistentFiles = (files, directory) => checkFiles('file', files, dir
 
 exports.getFileContent = filePath =>
   fs.readFileSync(exports.getTestDirectory(filePath), { encoding: 'utf-8' });
+
+exports.getCommands = () => {
+  require('dotenv').config();
```

Comment:
`dotenv` is only for development environments

Reply:
if I don't add it I get error locally, when I will run the tests and why I don't find the environment variable DEPLOYED!


If you recommend any improvement, I'd appreciate it.

Reply:
You could do this configuration in a hook like `beforeAll`, because maybe many tests needs environments variables in the future